addons/mass_mailing/views/mass_mailing_templates.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="mass_mailing.mass_mailing_kpi_link_trackers" name="Email Marketing: mailing link trackers statistic">
```

Comment:
Can be put in data (qweb templates used to contain html parts are generally put in data, not in views as they are not views displayed by the client or in a portal).